docs/01-overview/02-main-areas/application-connectivity/ac-08-access-ac-on-local.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Access Application Connector on a local Kyma deployment
+---
+
+To access Application Connector on a local deployment of Kyma, you must add the Kyma server certificate to the trusted certificate storage of your programming environment. This is necessary to connect the external solution to your local Kyma deployment, and allow for client certificate exchange and API registration.
```

Comment:
Our [guidelines](https://github.com/kyma-project/community/blob/main/guidelines/content-guidelines/08-style-and-terminology.md#terminology) state to use `must` instead of `should`, and this is something that's required and emphasized here, hence the `must`. 
I'd like it to stay :)